test/tests/morebits.date.js

```diff
@@ -0,0 +1,22 @@
+QUnit.module('Morebits.date');
+var now = Date.now();
+var timestamp = '09:31, 17 April 2020 (UTC)';
+QUnit.test('Construction', assert => {
+	assert.equal(new Morebits.date().getTime(), new Date().getTime(), 'Basic constructor');
```

Comment:
This failed for me the first time I ran it (off by one 1 millisecond), but worked fine on every later run.

Reply:
I initially had this with ISOString for exactly this reason (some browsers fudge milliseconds for security reasons), so maybe that would be better?

Reply:
I suppose we could lop off the final digit, since we don't really care.  It's reliably under 0.1% of times for me, so I'm fine leaving as-is if you are.